1.8/usage/service-discovery/third-party-solution/linkerd/index.md

```diff
@@ -0,0 +1,20 @@
+---
+post_title: linkerd
+menu_order: 0
+---
+
+linkerd is a service mesh for DC/OS. It is installed on every agent, and acts as a transparent, load-balancing proxy, providing service discovery and resilient communication between services.
+
+The linkerd Universe package is configured to use Marathon for service discovery. This means that applications and services can refer to each other by their Marathon task name. For example, a connection to `http://myservice` made through linkerd will be load-balanced over instances of the Marathon application `myservice`, without using DNS. linkerd can also be configured to use dedicated service discovery systems such as ZooKeeper, Consul or etcd (as well as DNS itself) and to failover between these systems and/or Marathon.
```

Comment:
- Change `linkerd can also be configured` to `You can also configure linkerd`
- Change `systems and/or Marathon` to `systems and Marathon` (https://americaneditor.wordpress.com/2013/03/11/the-drama-of-and-and-or/)